front/app/containers/Admin/projects/project/analysis/Tags/TagActions.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+
+import {
+  Dropdown,
+  DropdownListItem,
+  colors,
+  Spinner,
+  Text,
+  Icon,
+  Box,
+  IconButton,
+} from '@citizenlab/cl2-component-library';
+import useDeleteAnalysisTag from 'api/analysis_tags/useDeleteAnalysisTag';
+import { useIntl } from 'utils/cl-intl';
+import { useParams } from 'react-router-dom';
+import messages from '../messages';
+import RenameTagModal from './RenameTagModal';
+import Modal from 'components/UI/Modal';
+import { ITagData } from 'api/analysis_tags/types';
+import useAddAnalysisBulkTagging from 'api/analysis_taggings/useAnalysisBulkTaggings';
+import useAnalysisFilterParams from '../hooks/useAnalysisFilterParams';
+
+const TagActions = ({ tag }: { tag: ITagData }) => {
+  const [renameTagModalOpenedId, setRenameTagModalOpenedId] = useState('');
```

Comment:
Maybe `renameModalOpenedTagId` is clearer? It wasn't immediately obvious to me (from the name) that this stores a tagId.